src/js/stores/AppFormTransforms.js

```diff
@@ -1,9 +1,32 @@
+var Util = require("../helpers/Util");
+
+function trimDuplicableRows(rows) {
+  var trimmedRows = rows.map((row) => {
+    return Object.keys(row).reduce((memo, key) => {
+      if (row[key] != null
+        && key !== "consecutiveKey"
+        && !Util.isEmptyString(row[key])) {
+        memo[key] = row[key];
+      }
+      return memo;
+    }, {});
+  }).filter((row) => Object.keys(row).length);
```

Comment:
Yes or we would always have an empty array, for example

`container.docker.parameters: []`